.github/workflows/ci.yml

```diff
@@ -68,9 +68,9 @@ jobs:
       - name: Test 2.x
         if: ${{ !startsWith(matrix.scala, '3.') }}
         run: ./sbt ++${{ matrix.scala }} zio-ftp/test
-#      - name: Test 3.x
-#        if: ${{ startsWith(matrix.scala, '3.') }}
-#        run: ./sbt ++${{ matrix.scala }}! testDotty
+      - name: Test 3.x
+        if: ${{ startsWith(matrix.scala, '3.') }}
+        run: ./sbt ++${{ matrix.scala }}! testDotty
```

Comment:
use test

Reply:
Hope It'll work now.
I don't think there's a way for me to check whether the command will work without you first triggering the github ci build is there?

Reply:
you can test locally. :) 
i dont understand why i need to approve multiple times

Reply:
Yes, I did test locally :)
I was just a bit unsure/unfamiliar with the notation in this ci file.